Extended version of this patch was sent to mail address in bsod-0.1/readme.txt

--- Makefile
+++ Makefile
@@ -1,15 +1,15 @@
 VERSION=0.1
 
 DEBUG=-g -pedantic #-pg #-fprofile-arcs
-LDFLAGS=-lncurses $(DEBUG)
-CFLAGS+=-O2 -Wall -DVERSION=\"$(VERSION)\" $(DEBUG)
+LDLIBS=-lncurses
+CFLAGS+=-Wall -DVERSION=\"$(VERSION)\"
 
 OBJS=bsod.o
 
 all: bsod
 
 bsod: $(OBJS)
-	$(CC) -Wall $(OBJS) $(LDFLAGS) -o bsod
+	$(CC) $(LDFLAGS) $(CFLAGS) $(OBJS) $(LDLIBS) -o bsod
 
 install: bsod
 	cp bsod $(DESTDIR)/usr/local/bin
